tt_metal/host_api.hpp

```diff
@@ -127,6 +127,24 @@ KernelHandle CreateKernel(
     const std::variant<CoreCoord, CoreRange, CoreRangeSet> &core_spec,
     const std::variant<DataMovementConfig, ComputeConfig, EthernetConfig> &config);
 
+/**
+ * Creates a data movement kernel with no compile time arguments and adds it to the program.
```

Comment:
Can you update this comment. The API accepts compute or data movement kernels and compile time args are accepted via the `*Config`

You will have to add this API to `docs/source/tt-metalium/tt_metal/apis/host_apis/kernels/CreateKernel.rst`